### PR TITLE
Update the Starter Kits list

### PR DESCRIPTION
**Pull request by lvarayut**
I added a [relay-fullstack](https://github.com/lvarayut/relay-fullstack) project to the starter kits list.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,7 @@ Awesome resources for [Relay](https://github.com/facebook/relay), based on the [
 - [Relay Skeleton](https://github.com/fortruce/relay-skeleton) - React, Relay, GraphQL project skeleton.
 - [Relay Starter Kit](https://github.com/relayjs/relay-starter-kit) - An app that it already set up with a basic setup. Just clone and tweak to suit your needs!
 - [Simple Relay Starter](https://github.com/mhart/simple-relay-starter) - A Browserify version of the [Relay Starter Kit](https://github.com/relayjs/relay-starter-kit).
+- [Relay Fullstack](https://github.com/lvarayut/relay-fullstack) - Relay Starter Kit integrated with Relay, GraphQL, Express, ES6/ES7, JSX, Webpack, Babel, Material Design Lite, and PostCSS.
```
Comment (expede) [inline README.md]: This is super nit-picky, but some of this stuff can be assumed. Relay requires GraphQL, for instance. I'm open to the current version, but how about something along the lines of:

> The [Relay Starter Kit](https://github.com/relayjs/relay-starter-kit), integrated with Relay, [Express](http://expressjs.com/), [Material Design Lite](https://www.getmdl.io/), [PostCSS](https://github.com/postcss/postcss), Babel, and built with Webpack

**Comment by expede** [comment]
Nice kit! Just the one thought, but I'm giving this a :+1: Let me know when you're happy to have it merged, @lvarayut

**Comment by lvarayut** [comment]
Thanks for your feedback. Even though, Relay requires GraphQL in the current version, however, the developer team is working on extending to provide an interface to interact with in-memory and native device data as well. Here is the long [discussion](https://github.com/facebook/relay/issues/114) and also mentioned in the [official website](http://facebook.github.io/react/blog/2015/08/11/relay-technical-preview.html#whats-next).

Relay and React could also be written using any programming languages, such as TypeScript, and JSX is totally optional. 

That's why I would like to explicitly mention all of them in order to make it clear.

**Comment by expede** [comment]
@lvarayut Thanks for the links :smile: It looks like most of the proposed solutions involve GraphQL (or at least a GQL-compatible syntax). Such changes are uncertain enough that I think that we should cross that bridge once the changes are finalized, but I'm not really that fussed by it. I'm still a little weary of listing all of the internal packages, as if those change, we'll have to keep updating the description in this repo, which could be easy to forget.

All of that said, I also don't want this to turn into a bike shed :bike::house:. If you rebase away the merge conflicts, I'll merge the PR. Just let me know when it's good to go! :smile:

**Comment by lvarayut** [comment]
I modified it as you suggested.

**Comment by expede** [comment]
@lvarayut awesome; thanks :smile:

Also, thanks for creating Relay Fullstack: it's going to save me a lot of time :tada::+1:

**Comment by lvarayut** [comment]
Thanks for merging and glad you :heart: it. :smile:

**Pull request merged**